vita3k/app/src/app_init.cpp

```diff
@@ -193,6 +210,7 @@ bool init(EmuEnvState &state, Config &cfg, const Root &root_paths) {
     }
 
     LOG_INFO("Base path: {}", state.base_path.string());
+    LOG_INFO("Static assets path: {}", state.static_assets_path.string());
```

Comment:
This is confusing. Perhaps Shared assets below should just drop the "assets" part?

Reply:
fixed